docs/rst/tutorials/core/deployment/user_topics_multicast/user_topics_multicast.rst

```diff
@@ -0,0 +1,108 @@
+.. _tutorials_core_deployment_user_topics_multicast:
+
+Enabling multicast communication
+================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+.. _tutorials_core_deployment_user_topics_multicast_background:
+
+Background
+----------
+
+.. note::
+
+   This documentation assumes basic knowledge of UDP/TCP/IP concepts, namely unicast and multicast.
```

Comment:
A reference here would be nice in case of the lack of the basic knowledge

Reply:
I wouldn't know what to reference. Wikipedia seems a bit "cheesy" IDK. I think they can google